src/app/globals.css

```
@@ -1,9 +1,12 @@
 @import "tailwindcss";
 
-@layer base {
-  body {
-    background-color: #f8fafc;
-    color: #0f172a;
-    font-family: 'Inter', system-ui, -apple-system, sans-serif;
-  }
+@keyframes floatToast {
```

Stylelint (17.9.0)

[error] Expected keyframe name "floatToast" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)